cli/command/plugin/install_test.go

```diff
@@ -0,0 +1,142 @@
+package plugin
+
+import (
+	"fmt"
+	"io"
+	"io/ioutil"
+	"strings"
+	"testing"
+
+	"github.com/docker/cli/internal/test"
+	"github.com/docker/cli/internal/test/notary"
+	"github.com/docker/docker/api/types"
+
+	"gotest.tools/assert"
+	is "gotest.tools/assert/cmp"
+)
+
+func TestInstallErrors(t *testing.T) {
+	testCases := []struct {
+		description   string
+		args          []string
+		expectedError string
+		installFunc   func(name string, options types.PluginInstallOptions) (io.ReadCloser, error)
+	}{
+		{
+			description:   "insufficinet number of arguments",
```

Comment:
typo: insufficient

Reply:
Thank you for the review. I will update the PR to address this.